apps/condo/domains/meter/utils/taskSchema/AbstractMetersImporter.js

```diff
@@ -176,13 +176,26 @@ class AbstractMetersImporter {
                             if (messageForUser) {
                                 rowErrors.push(messageForUser)
                             } else {
-                                const mutationErrorMessages =
+                                let mutationErrorMessages =
                                     get(mutationError, ['originalError', 'errors', 0, 'data', 'messages'], []) || []
+
+                                // errors thrown to mutation from models
+                                if (mutationErrorMessages.length === 0) {
+                                    mutationErrorMessages = (get(mutationError, ['originalError', 'errors', 0, 'originalError', 'errors'], []) || []).reduce((result, error) => {
```

Comment:
Why do you need another default value as empty array. get function already had an fallback value

Reply:
null is valid value for `get()`, but I need an array here.
<img width="350" alt="image" src="https://github.com/user-attachments/assets/f5e1e55b-e3fa-4ec3-bb87-c2bd7fd72c6e">